bash/users.sh

```diff
@@ -0,0 +1,263 @@
+#!/bin/bash
+
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy 
+# of this software and associated documentation files (the "Software"), to 
+# deal in the Software without restriction, including without limitation the 
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or 
+# sell copies of the Software, and to permit persons to whom the Software is 
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in 
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR 
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE 
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING 
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+# DEALINGS IN THE SOFTWARE.
+
+echo "Example script using the user's API"
+echo
+# Check cURL command if available (required), abort if does not exists
+type curl >/dev/null 2>&1 || { echo >&2 "Required curl but it's not installed. Aborting."; exit 1; }
+echo
+
+# Make sure Variables are encapsulated with "" if there are spaces within them
+uuid='<Your UUID>'
```

Comment:
why did you not make these command line parameters?

Reply:
I could. Based on my understanding these scripts are supposed to be examples on how it _could_ be done, but not an exact this is how it _has_ to be done. I have a planning meeting tomorrow to see if this is something we want explicit or leave more open.

Reply:
I will understand and approve if you say that user experience requires this and not cli args ;-)

However, I like what @adam-poppenhagen did where he left in the commented out the cli args as a possibility for the user.  If the UX is supposed to be withOUT cli args though then what you have will suffice!